activemodel/test/cases/validations/with_validation_test.rb

```diff
@@ -145,4 +145,12 @@ def check_validity!
     assert_empty topic.errors[:title]
     assert_equal ["is missing"], topic.errors[:content]
   end
+
+  test "validates_with can use multiple 'ActiveModel::EachValidator's" do
+    Topic.validates_with(ValidatorPerEachAttribute, ValidatorPerEachAttribute, attributes: [:title])
+    topic = Topic.new
+    assert_not_predicate topic, :valid?
+    assert_not_empty topic.errors[:title]
+    assert_equal ["Value is ", "Value is "], topic.errors[:title]
+  end
```

Comment:
If we fix this for all `validates_with` calls, we can change this test to assert that the given options are not mutated (instead of asserting specifically that multiple `EachValidator`s work).